src/sbt-test/skip-sbt1.4/testkit/build.sbt

```diff
@@ -1,7 +1,7 @@
 lazy val V = _root_.scalafix.sbt.BuildInfo
 
 lazy val rulesCrossVersions = Seq(V.scala213, V.scala212)
-lazy val scala3Version = "3.0.0"
+lazy val scala3Version = "3.3.1"
```

Comment:
`sbt-1.5/*` was ignored on JDK21 until now. The fix was simply to get a compatible scala 3 version (per the [compatibility matrix](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html)).